lib/scripts/generate-graphql-types.js

```diff
@@ -8,18 +8,40 @@ const { execSync } = require('child_process'); // Execute shell commands
 function generateGraphQLTypes() {
   console.log('Starting generation of TypeScript types...');
 
-  // Delete current types file
+  // Backup and delete current types file if exists
   const TYPES_PATH = 'lib/graphql/types.ts';
-  if (fs.existsSync(TYPES_PATH)) {
-    fs.unlinkSync(TYPES_PATH);
+  const BACKUP_TYPES_PATH = 'lib/graphql/types-backup.ts';
+
+  if (existsSync(TYPES_PATH)) {
+    // Backup file
+    console.log(`Backing up ${TYPES_PATH}...`);
+    copyFileSync(TYPES_PATH, BACKUP_TYPES_PATH);
+
+    // Delete file
+    unlinkSync(TYPES_PATH);
   }
 
-  // Codegen TypeScript types
-  console.log('Generating TypeScript types 🛠️');
-  execSync('graphql-codegen --config codegen.yml');
-  console.log('TypeScript types file created at lib/graphql/types.ts ✅\n');
+  try {
+    // Codegen TypeScript types
+    console.log('Generating TypeScript types 🛠️');
+    execSync('graphql-codegen --config codegen.yml', { stdio: 'inherit' });
+    console.log(`TypeScript types file created at ${TYPES_PATH} ✅\n`);
+
+    // Delete backup if it was made
+    if (existsSync(BACKUP_TYPES_PATH)) {
+      unlinkSync(BACKUP_TYPES_PATH);
+    }
 
-  console.log('DONE ✅');
+    console.log('DONE ✔️');
```

Comment:
Just wondering why you changed checkmarks? Personally I liked the green checkmark better since its a lot easier to see in a black terminal but it doesn't really matter 😅

Reply:
Just wanted to make it consistent with the red X for failure - will change it back though, kinda liked the original one more too xD